README.md

```diff
@@ -54,7 +57,7 @@
 
 **Note: If the installation path and application name are the same, it will lead to conflict ([#142](https://github.com/lencx/ChatGPT/issues/142#issuecomment-0.11.0))**
 
-### Mac
+### Mac:
```

Comment:
This contradicts Markdown lint rule MD026 - https://github.com/DavidAnson/markdownlint/blob/main/doc/md026.md

Headings should really not end with punctuation!